### PR TITLE
new structure and chunk mappings

### PR DESCRIPTION
**Pull request by eristhea**
redo of #322, minus the absolutely baffling build failures this time

notes:
fixes some bad names for registry keys

### CODE REVIEW

**Review by OroArmor — DISMISSED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/structure/piece/StructurePieceType.mapping [hunk 1]
```diff
@@ -67,6 +67,9 @@ CLASS net/minecraft/unmapped/C_etqvykma net/minecraft/structure/piece/StructureP
 		ARG 0 simplePieceType
 		ARG 1 id
 	CLASS C_blzadryc ManagerAware
+		METHOD load load (Lnet/minecraft/unmapped/C_zxhjfyir;Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
+			ARG 1 structureTemplateManager
+			ARG 2 nbt
 	CLASS C_cdoppjru Simple
-		METHOD load (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
+		METHOD load load (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
```
Comment (EnnuiL) [inline mappings/net/minecraft/structure/piece/StructurePieceType.mapping:74]: hm...?

File: mappings/net/minecraft/world/storage/VersionedChunkStorage.mapping [hunk 2]
```diff
@@ -2,12 +2,21 @@ CLASS net/minecraft/unmapped/C_hzmyuskp net/minecraft/world/storage/VersionedChu
 	FIELD f_aklywswt featureUpdater Lnet/minecraft/unmapped/C_qpbbnpoq;
 	FIELD f_jnzhwftu dataFixer Lcom/mojang/datafixers/DataFixer;
 	FIELD f_rmprhtbt worker Lnet/minecraft/unmapped/C_ffgsaknh;
+	METHOD <init> (Ljava/nio/file/Path;Lcom/mojang/datafixers/DataFixer;Z)V
+		ARG 1 directory
+		ARG 2 dataFixer
+		ARG 3 dsync
```
Comment (EnnuiL) [inline mappings/net/minecraft/world/storage/VersionedChunkStorage.mapping:8]: This is a moment where I wish there was a diff here :P
I'm not sure about the convention around here, but yeah, for me, `dsync` does look like a weird name...?

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Looks good! Just few things

File: mappings/net/minecraft/structure/piece/StructurePieceType.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/structure/piece/StructurePieceType.mapping:74]: just to show that it's mapped :p

File: mappings/net/minecraft/world/storage/VersionedChunkStorage.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/world/storage/VersionedChunkStorage.mapping:8]: `dsync` is a boolean that's passed straight into another method as a parameter named `dsync`, so I simply copied that name

File: mappings/net/minecraft/structure/piece/StructurePieceType.mapping (hunk 1 above)
Comment (EnnuiL) [reply mappings/net/minecraft/structure/piece/StructurePieceType.mapping:74]: oh, i'm pretty sure that Enigma does have a `Mark as Unobfuscated` feature, but yeah, maybe it does exactly that...?

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**